ui/src/app/cron-workflows/components/cron-workflow-list/cron-workflow-list.tsx

```diff
@@ -137,8 +138,10 @@ export class CronWorkflowList extends BasePage<RouteComponentProps<any>, State>
                             <div className='columns small-1'>{w.spec.suspend ? <i className='fa fa-pause' /> : <i className='fa fa-clock' />}</div>
                             <div className='columns small-3'>{w.metadata.name}</div>
                             <div className='columns small-2'>{w.metadata.namespace}</div>
-                            <div className='columns small-2'>{w.spec.schedule}</div>
-                            <div className='columns small-2'>
+                            <div className='columns small-3'>
+                                <Schedule schedule={w.spec.schedule} />
```

Comment:
I don't think the "pretty" schedule should be displayed in the list view. It's not necessary when looking at all cron workflows (as opposed to just one in detail) and from your screenshot it looks like they're frequently cut off (bad UX)

Reply:
I'd like this to be visible in the list. My reasons:

* Commonly I believe that people use simple schedules that are short.
* It is easier for users to visually parse, vs raw schedule.
* I expect truncation to happen about. 

More complex example:

![image](https://user-images.githubusercontent.com/1142830/107684955-4544f100-6c58-11eb-858c-5b3ac9225024.png)

I'm going to ask the community.

Reply:
I can see a bug in the library. `0 12 * */1 *`. This should be "At 12:00 in every month.”

Reply:
huh - 1st of the month?

Reply:
I *think* "*/1" === "*"?